data-explorer/kusto/query/mvexpandoperator.md

```diff
@@ -45,7 +45,9 @@ Two modes of property-bag expansions are supported:
   allowing uniform access to keys and values (also, for example, running a distinct-count aggregation
   over property names). 
 
-**Examples**
+## Examples
```

Comment:
@orspod - not sure this is aligned with the H2/H3 model of the docs

Reply:
Looking at https://docs.microsoft.com/en-us/azure/data-explorer/ingest-data-overview, I believe what I've put in will not give the results I was expecting.  I double check with other docs (AKS) and it is consistent.

It seems that whatever is rendering the doc only takes the H2 headers to put them in the right-hand side navigation.  There is no hierarchical (I thought I remember that but can't find an example).

My goal here was to get better separation between examples so that a customer can come in and quickly find the example they are looking for.  I think the current H2/H3 kind of do that as headers are clearly separating each example.  They will not appear on the right-hand side thought.

Taking inspiration from other docs:

https://docs.microsoft.com/en-us/azure/cosmos-db/sql-query-where#examples
https://docs.microsoft.com/en-us/sql/t-sql/queries/select-over-clause-transact-sql?view=sql-server-ver15#examples

They seem to essentially do H2/H3.

Your thoughts?

Reply:
@vplauzon, I talked to @YaelSchuster (in our content writing team) and she says ## is ok, as long as there's one more ##. Since we don't have one on this page, she suggested to change `**See also**` to `## See also`

Reply:
@vplauzon ,@sloutsky, @slavikn84 - if we want the sections within the "examples" to show up in the right-hand "doc TOC" they need to be H2s. This is something we are currently grappling with since the query docs use bold formatting to separate out the different sections. Since some of these sub-topics aren't "Examples", we may be able to change some of them to H2 headings?

Reply:
> @orspod - not sure this is aligned with the H2/H3 model of the docs

@sloutsky - I agree. I think we need to discuss further and determine if we want to alter the current model of the ref docs to an H2/H3 model?

Reply:
@orspod, so if I understand your comment, the right-hand side isn't a hierarchy.  It displays the h2 only.  Right?

Reply:
The two examples I linked to use only ## Examples, then ### for each example (actually, the SQL one has a second ## for Parallel Data Warehouse, so kind of categorizing the examples).  They aren't displayed on the right but they are quickly identifiable when scrolling down.  That would be a plus IMO.

Reply:
@vplauzon - yes, unfortunately the right-hand side only displays H2s